data/category/base/data.json

```diff
@@ -1,25 +1,52 @@
 {
-    "data": {
-        "category-dairy": {
-            "name": "Dairy"
-        },
-        "category-baking" : {
-            "name": "Baking"
-        },
-        "category-produce" : {
-            "name": "Produce"
-        },
-        "category-bakery" : {
-            "name": "Bakery"
-        },
-        "category-deli" : {
-            "name": "Deli"
-        },
-        "category-frozen" : {
-            "name": "Frozen"
-        },
-        "category-canned" : {
-            "name": "Canned"
-        }
+  "data": {
+    "category-dairy": {
+      "name": "Dairy"
+    },
+    "category-baking": {
+      "name": "Baking"
+    },
+    "category-produce": {
+      "name": "Produce"
+    },
+    "category-bakery": {
+      "name": "Bakery"
+    },
+    "category-deli": {
+      "name": "Deli"
+    },
+    "category-frozen": {
+      "name": "Frozen"
+    },
+    "category-canned": {
+      "name": "Canned"
+    },
+    "category-fish": {
+      "name": "Fish"
+    },
+    "category-meat": {
+      "name": "Meat"
+    },
+    "category-herbs-spices": {
+      "name": "Herbs and Spices"
+    },
+    "category-bread-cereal": {
+      "name": "Bread and Cereals"
+    },
+    "category-oil-vinegar": {
+      "name": "Oil and Vinegar"
+    },
+    "category-condiments-sauces": {
+      "name": "Condiments and Sauces"
+    },
+    "category-cheese": {
+      "name": "Cheese"
+    },
+    "category-dry": {
+      "name": "Dry Goods"
+    },
+    "category-asian": {
```

Comment:
We often summarize all spezial cuisines as "Feinkost" which roughly translates to delicacies, what do you think? Because supermarkets usually put all foods that are not "normal" for a region together. Hie is this handled elsewhere?

Reply:
Here in the US, you have a large number of regions. Some areas may have a "Ethnic foods" area that contains all asian, mexican, etc. Some areas may have a dedicated mexican section and maybe a dedicated asian area. Some areas have dedicated "tortillas" and "beans" sections where there are more options than would reasonably just fit into a "mexican" section.

Reply:
Whoops, forgot to make a comment on the asian section. I have that for all of the things I get at the asian food store: nori, kewpie mayo, mirin, ponzu, 500ml tubs of curry paste, white/brown sweet potatoes, chinese broccoli and so on. Things you typically can't get at the regular grocery.
Could imagine having this as my personal customization and not part of the regular dataset.

Reply:
This PR is still a great help as we work on shaping what we want to see this dataset become! 

For your specific food from the asian store, I would suggest setting these ingredients to be bought at a specific store, rather than trying to use supermarket categories to decide what store to buy at. Are you familiar with that capability?

Reply:
TIME TO EAT MY OWN WORDS! lololol. Sorry about that. I am so used to my method where I have categories with the store named in them. My foods are nicely assigned to the store specific categories. As such, I was wrong in talking about assigning foods to specific stores. I maintain 2 fruits categories. One is named `Fruits (Costco)` for bulk fruits and one is named `Fruits (Winco)` for small scale purchase. Similar for `Meat (Costco)` and `Meat (Winco)`. 

I ONLY buy ground beef from Costco, so that's what its assigned to. But buying hamburgers, I only do it at Winco.

Reply:
very interesting. One thing we also always have to consider is that those basic categories are universal around the world whereever someone uses tandoor, so someone using it in asia might be confused by a category "asian" on the other hand "specialty" is also just normal depending on where you are. 

I guess having special categories for the major couisines around the world might make sense, altough we should have these regions rather large. I am not entirely sure but what do you think about these:

- North American 
- South American (is this even a major couisine?)
- Central American (which would include Mexico?) .. 
- Easter European
- Western European (altough french and italian food is similar but german is completely different but i might be biased since i am from germany)
- Eastern European 
- Asian (i am sure someone with knowledge of the asian couisine would hang me for putting it all into one category)

Should we maybe change the datatype on the food to more easily support multiple categories per food in the base langauge file similar to the preferred shopping unit?

maybe

```
supermarket_category: {
 "base": "specialty",
"de": "XYZ",
"us": "ABC",
}
```

or have it as an array and use the first one thats found in the regional category list (the french one would not include french for example) but thats also very hard to define without lots of knowledge about the local supermarket strucutre. 

sorry for all the loose thoughts ...

Reply:
Hi, sorry for the late reply. I got bogged down with work stuff. 
I've reworked the categories and given them descriptions which should help make it clear what I meant. Of course we will need to add or split the categories as we add new ingredients, but it helps me to work from the categories inward.

I think we should go with a category named specialty, _if_ we ever come up with ingredients that need this category. When I lived in Brasil I had a real hard time getting any sort of specialty goods like asian stuff or even some spices. (Soy sauce would have been categorized under sauces there.) 

So I think it does not make sense to include a specialty ingredients category at all. Like you said, specialty is different in every region and I cannot think of an ingredient off the top of my head which would be a specialty ingredient in all regions.